File: Backend/garderie/garderie/src/main/java/com/example/garderie/dto/SeanceDTO.java
package com.example.garderie.dto;

import java.time.LocalTime;
import com.example.garderie.entities.Seance.RecurrenceType;

public record SeanceDTO(
        String nom,
        LocalTime horaireDebut,
        LocalTime horaireFin,
        Long classeId,
        Long enseignantId,
        boolean obligatoire,
        String lieu,
        String startDate,
        String endDate,
        RecurrenceType recurrenceType
) {}

File: Frontend/main/src/app/teacher/evaluation/evaluation-list/seance.dto.ts
// seance.dto.ts
export interface SeanceDTO {
  nom: string;
  horaireDebut: string; // Format ISO "HH:mm"
  horaireFin: string;   // Format ISO "HH:mm"
  obligatoire: boolean;
  lieu: string;
  date: string;         // Format ISO "YYYY-MM-DD"
}
